modules/virt-sharing-custom-vm-template.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+:_content-type: PROCEDURE
+[id="virt-sharing-custom-vm-template_{context}"]
+= Sharing a custom virtual machine template
+
+Share virtual machine templates to the public {product-title} namespace so that you minimize creating multiple templates in private namespaces.
+
+.Prerequisites
+
+* You must have cluster administrator privileges to share virtual machine templates to the public {product-title} namespace.
```

Comment:
You can't use the OCP attribute when you are talking about the "OpenShift namespace". You need to spell it out.